src/order/hom/basic.lean

```diff
@@ -647,6 +650,23 @@ have gf : ∀ (a : α), a = g (f a) := by { intro, rw [←cmp_eq_eq_iff, h, cmp_
   right_inv := by { intro, rw [←cmp_eq_eq_iff, ←h, cmp_self_eq_eq] },
   map_rel_iff' := by { intros, apply le_iff_le_of_cmp_eq_cmp, convert (h _ _).symm, apply gf } }
 
+/-- To show that `f : α →o β` and `g : β →o α` make up an order isomorphism it is enough to show
+    that `g` is the inverse of `f`-/
+def of_hom_inv {F : Type u_1} {G : Type u_4} [order_hom_class F α β] [order_hom_class G β α]
+  {f : F} {g : G} (h₁ : (f : α →o β).comp (g : β →o α) = order_hom.id)
+    (h₂ : (g : β →o α).comp (f : α →o β) = order_hom.id) : α ≃o β :=
```

Comment:
I think using `order_hom_class` here is probably a mistake. Do you agree, @YaelDillies?

Reply:
In that case,
```suggestion
def of_hom_inv {f : α →o β} {g : β →o α} (h₁ : f.comp g = order_hom.id) (h₂ : g.comp f = order_hom.id) : α ≃o β :=
```

Reply:
I would not claim that there is a convention either way yet.

Reply:
Well, we have only two other definitions called `of_hom_inv`, and they're implemented with `*_hom_class`, so I think in the corner we're currently in there is definitely a convention

Reply:
Yes ti should be true, but note that the pattern is usually that the equiv is stronger than a bunch of homs (`order_iso` implies all current order homs, for example), so there should rarely be a need for the stronger version.

Reply:
Ahh yes. I think it should be funlike then.

Reply:
Ok I've made the homs explicit !